addons/sale_timesheet/__init__.py

```diff
@@ -10,7 +10,11 @@ def uninstall_hook(env):
     env.ref("account.account_analytic_line_rule_billing_user").write({'domain_force': "[(1, '=', 1)]"})
 
 def _sale_timesheet_post_init(env):
-    products = env['product.template'].search([('detailed_type', '=', 'service'), ('invoice_policy', '=', 'order'), ('service_type', '=', 'manual')])
+    products = env['product.template'].search([
+        ('type', '=', 'service'),
+        ('invoice_policy', '=', 'order'),
+        ('service_type', '=', 'manual'),
+    ])
```

Comment:
Mhm, seems wrong, event booth, tickets, booking fees, ... will be set as `timesheet` here :thinking: